symfony/src/Controller/MeasurementsController.php

```diff
@@ -473,7 +473,7 @@ public function measurementFhirAction($participantId, $measurementId, Request $r
         }
         $fhir = $measurement->getFhir($date, $parentRdrId);
         if ($isTest) {
-            $fhir = \Tests\Pmi\Evaluation\EvaluationTest::getNormalizedFhir($fhir);
+            $fhir = \App\Test\Entity\MeasurementTest::getNormalizedFhir($fhir);
```

Comment:
Note that this still appears in the `phpstan.baseline` file as still un-callable.